modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectLayoutLocalServiceImpl.java

```diff
@@ -567,6 +576,44 @@ private void _validate(
 		}
 	}
 
+	private void _validateObjectLayoutBoxCategorizationType(
```

Comment:
Hi @seltong, I believe that you can use the _validate method to check the restricions of categorization objectLayoutBox.